
# observation 
pixel_count: 2

init_b: 0.0

A_a: 0.0
A_b: 0.9

v_sigma_a: 0.000001
v_sigma_b: 0.5

sim_w_sigma: 0.001
w_sigma: 0.001

w_sigma_b: 0.4

max_cycles: 100

cycle_of_error: 10
error_pixels: 1
error_duration: 60
error_value: 10.0
const_error: true
gradual_error: false
gradual_delay: 1

factorize: true
print_states: false

